node/pkg/cni/token_watch_test.go

```diff
@@ -0,0 +1,90 @@
+package cni_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/projectcalico/calico/node/pkg/cni"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+var _ = Describe("FV tests", func() {
+	const serviceAccountName = "cni-token-watch-test"
+	const namespace = metav1.NamespaceSystem
+
+	It("should create a token successfully", func() {
+		clientset := createKubernetesClient()
+		setupServiceAccount(clientset, serviceAccountName, namespace)
```

Comment:
Would probably be nicer to have these in a BeforeEach and AfterEach to remove code duplication.